### DISCUSSION TITLE
Why would I have a missing character in my PDF?

### DISCUSSION THREAD

**Discussion by drusellers** [QuestPDF/QuestPDF#597] [Q&A]
Hello 👋,

I'm generating a PDF and when I do so locally on my Mac I get a correctly rendered `$` sign. But when I upload my application to my Ubuntu server, and use ASP.Net to then stream the bytes produced down I get a missing `¤` char for the `$`

<img width="78" alt="CleanShot 2023-06-26 at 12 50 35@2x" src="https://github.com/QuestPDF/QuestPDF/assets/63355/a5b982c0-8583-4f28-ab2e-ca0f87153ac6">

I feel like this is me missing something really obvious because I'm new to PDF generation.

TY

**Comment by MarcinZiabek** [comment]
Hi 😁 

Would you please share a minimal example showing this issue? I want to investigate it before providing any guidance. There is no real reason for similar behavior if you use the same font file. Thank you.

**Reply by drusellers** [reply]
It ended up being that I was running my app in `Globalization Invariant Mode` once I fixed that issue and supplied a CultureInfo the problem resolved itself.

https://github.com/dotnet/runtime/blob/main/docs/design/features/globalization-invariant-mode.md
